src/css-validator/index.js

```diff
@@ -75,34 +75,36 @@ var validateSelectors = function(bundle, result) {
 
 };
 
-var validate = function(bundle, css, cb) {
+var validate = function(bundle, css) {
 
-    if (!shouldValidate(bundle)) {
-        cb();
-        return;
-    }
-
-    var stats = new StyleStats(css, cssStatsSettings);
+    return new Promise(function(resolve, reject) {
 
-    stats.parse(function onCssStatsParsed(err, result) {
-
-        // If it's an error about an empty stylesheet, don't throw.
-        // This occurs on new bundles that have no styles yet.
-        // Otherwise, bubble the error up.
-        if (err && err.message !== 'Rule is not found.') {
-            cb(err);
+        if (!shouldValidate(bundle)) {
+            resolve(css);

```

Comment:
I updated this to pass through the CSS code so we can use it in the next stage of the promise chain.